src/controller/brake.c

```diff
@@ -18,25 +18,25 @@
 
 void brake_init (void)
 {
-  //brake pin as external input pin interrupt
+  // brake pin as external input pin interrupt
   GPIO_Init(BRAKE__PORT, BRAKE__PIN, GPIO_MODE_IN_FL_IT); // with external interrupt
 
-  //initialize the Interrupt sensitivity
+  // initialize the interrupt sensitivity
   EXTI_SetExtIntSensitivity(EXTI_PORT_GPIOC, EXTI_SENSITIVITY_RISE_FALL);
 }
 
 
 // brake signal interrupt
 void EXTI_PORTC_IRQHandler(void) __interrupt(EXTI_PORTC_IRQHANDLER)
 {
-  if (brake_is_set())
+/*   if (brake_is_set())
```

Comment:
Are we sure removing all this brake related code is safe?
Are you running brakes on your test system ?

Reply:
WIP.